torch/onnx/symbolic_opset9.py

```diff
@@ -1512,13 +1512,16 @@ def full(g, sizes, value, dtype, layout, device, pin_memory=False):
                     value_t=torch.tensor([const_value], dtype=sym_help.scalar_type_to_pytorch_type[dtype]))
 
 
-@parse_args('v', 'f', 'i', 'v', 'v', 'v', 'v')
 def full_like(g, input, fill_value, dtype=None, layout=None, device=None, pin_memory=False, memory_format=None):
```

Comment:
We would at least want two tests in onnx_onnxruntime to cover export for both branches of this functions.
One for sym_help._is_value(fill_value) branch and one for the other branch.